requirements.txt

```diff
@@ -0,0 +1,17 @@
+git+https://github.com/arcalot/arcaflow-plugin-sdk-python@bce430a59a5782ef26aed9a196149031c667c6ff
```

Comment:
Is this really a good thing to do?  Can we make an official release of the plugin so we can use a tag?